package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "digital-analytics-program",
+  "version": "4.0.0",
```

Comment:
Just noting, this is something that e-NOR / @tdlowden will need to keep in sync with future releases.

Reply:
Understood. Thanks for pointing out

Reply:
hmmm. i had been only updating version numbers specifically when https://github.com/digital-analytics-program/gov-wide-code/blob/master/Universal-Federated-Analytics-Min.js and the unminified files change. when you say the code is different, you don't mean those files, correct? there are other changes that happen in this repo, e.g. in the documentation section, and I never consider it "a release". the "release" is, up until now, changes to the core DAP file.

Reply:
this also brings into focus the need for me to use the native `release` feature in GH

Reply:
When I do releases, I increment version number for even non-code changes... in this case, I consider adding package.json worthy of an increment since it gives developers completely new functionality.

Using semver (https://semver.org/), this could either be MINOR or PATCH update.

Also, I agree that you should use GH release function. I can help out with that if you like.